package.json

```diff
@@ -66,7 +72,8 @@
         "faker": "https://github.com/marak/Faker.js#v5.1.0",
         "jest": "^26.0.1",
         "jsdoc": "^3.6.4",
-        "prytaneum-typings": "https://github.com/c2c-project/prytaneum-typings#v1.0.0-alpha.78",
+        "papaparse": "^5.3.0",
+        "prytaneum-typings": "https://github.com/c2c-project/prytaneum-typings#v1.0.0-alpha.83",
```

Comment:
You can upgrade this to `https://github.com/c2c-project/prytaneum-typings#3ba38df`